streams/src/main/java/org/apache/kafka/streams/KeyQueryMetadata.java

```diff
@@ -50,32 +50,65 @@ public KeyQueryMetadata(final HostInfo activeHost, final Set<HostInfo> standbyHo
     }
 
     /**
-     * Get the Active streams instance for given key
+     * Get the active Kafka Streams instance for given key.
      *
      * @return active instance's {@link HostInfo}
+     * @deprecated Use {@link #activeHost()} instead.
      */
+    @Deprecated
     public HostInfo getActiveHost() {
         return activeHost;
     }
 
     /**
-     * Get the Streams instances that host the key as standbys
+     * Get the Kafka Streams instances that host the key as standbys.
      *
      * @return set of standby {@link HostInfo} or a empty set, if no standbys are configured
+     * @deprecated Use {@link #standbyHosts()} instead.
      */
+    @Deprecated
     public Set<HostInfo> getStandbyHosts() {
         return standbyHosts;
     }
 
     /**
-     * Get the Store partition corresponding to the key.
+     * Get the store partition corresponding to the key.
      *
      * @return store partition number
+     * @deprecated Use {@link #partition()} instead.
      */
+    @Deprecated
     public int getPartition() {
         return partition;
     }
 
+    /**
+     * Get the active Kafka Streams instance for given key.
+     *
+     * @return active instance's {@link HostInfo}
+     */
+    public HostInfo activeHost() {
+        return activeHost;
+    }
+
+    /**
+     * Get the Kafka Streams instances that host the key as standbys.
+     *
+     * @return set of standby {@link HostInfo} or a empty set, if no standbys are configured
+     */
+    public Set<HostInfo> standbyHosts() {
+        return standbyHosts;
+    }
+
+    /**
+     * Get the store partition corresponding to the key.
```

Comment:
Get the partition number corresponding to the key.

Reply:
Relatable to the comments from line 106 right ?